content/themes/biojs-theme/assets/css/main.css

```diff
@@ -92,6 +92,13 @@ a[href] {
     width: 0%;
     height: 100%; }
 
+nav {
```

Comment:
@ayusharma For the changes being available to other developers as well, please do not change the main.css file which is recompiled anytime someone uses our project's [grunt](gruntjs.com) task to edit css. Instead, do your changes in the sass directory only, as described in our [README](https://github.com/biojs/biojs.net2#get-started-with-sass). If you are not familiar with SASS yet, the [SASS lang Documentation](http://sass-lang.com/) is a great place to get started.